app/javascript/packs/pages/provider_signup/ProviderRequestForm.jsx

```diff
@@ -46,6 +46,10 @@ const providerRequestTypes = [
   {
     label: "Supplies",
     value: "supplies"
+  },
+  {
+    label: "Pet care",
+    value: "petCare"
   }
```

Comment:
Unsure of where in the checklist this item should live. Items are not sorted alphabetically.

Further, thoughts on a `value` of `petCare` vs `pets`?

Reply:
No strong opinion, we could do `pets` for simplicity.

Reply:
Changed 👍